Automattic/wp-calypso
# Remove the Help Center Survey

Reverts https://github.com/Automattic/wp-calypso/pull/82194

### Testing steps
1. Using the live link, create a new WP.com account.
2. Go to My Home.
3. Click the Help Center.
4. You shouldn't see the suvery.
